src/main/kotlin/com/openlattice/auditing/LocalAuditingService.kt

```diff
@@ -45,26 +44,25 @@ class LocalAuditingService(
                     if (auditEntitySetConfiguration.auditEdgeEntitySet != null) {
                         val auditEdgeEntitySet = auditEntitySetConfiguration.auditEdgeEntitySet
 
-                        val lm = ArrayListMultimap.create<UUID, DataEdge>()
+                        val dataEdges = mutableListOf<DataEdge>()
                         entityKeyIds.asSequence().zip(auditableEvents.asSequence())
                                 .filter { it.second.entities.isPresent }
                                 .forEach { (auditEntityKeyId, ae) ->
                                     val aeEntitySetId = ae.aclKey[0]
                                     val aeEntityKeyIds = ae.entities.get()
                                     aeEntityKeyIds.forEach { id ->
-                                        lm.put(
-                                                auditEdgeEntitySet,
+                                        dataEdges.add(
                                                 DataEdge(
                                                         EntityDataKey(aeEntitySetId, id),
                                                         EntityDataKey(auditEntitySet, auditEntityKeyId),
                                                         ImmutableMap.of()
                                                 )
                                         )
-                                        return@forEach
                                     }
                                 }
+                        val associations = mutableMapOf(auditEdgeEntitySet to dataEdges)
```

Comment:
same question as above (could be immutable?)

Reply:
Yep :)